src/business_logic/execution/error.rs

```diff
@@ -68,6 +67,8 @@ pub enum ExecutionError {
     OutOfBound,
     #[error("Calling other contracts during validate execution is forbidden")]
     UnauthorizedActionOnValidate,
+    #[error("Call to another contract has been done")]
+    OtherContractCalls,
```

Comment:
What do you think about calling it `InvalidContractCall` o `UnexpectedContractCall` ?